client/src/components/availability/SelectTimeForm.js

```diff
@@ -0,0 +1,105 @@
+import React, { useState, useContext }from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import {Button, TextField} from '@material-ui/core';
+import {format} from "date-fns";
+import { addAvailability } from "../../actions/availability";
+import { AuthDispatchContext, AuthStateContext } from "../../context/AuthContext";
+import AlertMessage from "../Alert";
+
+const useStyles = makeStyles((theme) => ({
+    container: {
+      display: 'flex',
+      flexWrap: 'wrap',
+      alignItems: 'center'
+    },
+    textField: {
+      margin: theme.spacing(1),
+      width: 200,
+    },
+  }));
+
+function AddTimeForm(props){
+    const selectedDate = format(props.selectedDate, "yyyy-MM-dd");
+    const classes = useStyles();
+    const [selectedFrom, setSelectedFrom] = useState("08:00");
+    const [selectedTo, setSelectedTo] = useState("16:00");
+    const [alert, setAlert] = useState({ error: false, message: "" });
+    const [addText, setAddText] = useState("ADD");
+    const [color, setColor] = useState("primary");
+    const [disabled, setDisabled] = useState(false);
+    const dispatch = useContext(AuthDispatchContext);
+    const { profile } = useContext(AuthStateContext);
+    
+    const handleFromChange = (event) => {
+        setAddText("ADD");
+        setColor("primary");
+        setDisabled(false)
+        setSelectedFrom(event.target.value)
+    }
+    const handleToChange = (event) => {
+        setAddText("ADD");
+        setColor("primary");
+        setDisabled(false);
+        setSelectedTo(event.target.value);
+    }
+    const handleSubmit = (event) => {
+        event.preventDefault();
+        setAddText("ADDING...");
+        if(!selectedDate){
+            setAlert({error:true, message: "Please Select Date!"});
+            setAddText("ADD");
+        }
```

Comment:
selectedDate is being checked to not be empty, but time is not. I presses delete button when chose time and it pass into db
![avail2](https://user-images.githubusercontent.com/47538460/105940131-9a61ef80-6028-11eb-85c2-42f37819c00a.jpg)